CuitService/CuitNumberValidationAttribute.cs

```diff
@@ -8,7 +8,7 @@ namespace CuitService
 {
     public class CuitNumberValidationAttribute : ValidationAttribute
     {
-        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+        protected override ValidationResult? IsValid(object? value, ValidationContext validationContext)
```

Comment:
```diff
-          protected override ValidationResult? IsValid(object? value, ValidationContext validationContext)
+          protected override ValidationResult IsValid(object? value, ValidationContext validationContext)

Reply:
The change was OK, https://github.com/FromDoppler/cuit-service/pull/88/files#r611075267